tests/integration/conditions_feature/test_condition_behavior.py

```diff
@@ -88,7 +88,7 @@ def flow(request, temp_workspace):
     return f
 
 
-@pytest.mark.parametrize('flow', ['python', 'yaml'], indirect=True)
```

Comment:
be careful with this, this test must work

Reply:
oups forgot to uncomment